home/.bin/create-hotfix

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env ruby
```

Comment:
We should add your documentation from the pull request as comments here.  With any luck, others will be able to use this script and (hopefully) contribute changes  back to us.